qualimap/qualimap.cwl

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env cwl-runner
+cwlVersion: v1.0
+class: CommandLineTool
+
+# Metadata
+label: qualimap-qc
+doc: This is qualimap CWL tool definition http://qualimap.bioinfo.cipf.es/
+
+hints:
+  ResourceRequirement:
+    ramMin: 4000
+    coresMin: 1
+  SoftwareRequirement:
+    packages:
+      qualimap:
+        version: [ "2.2.2d" ]
+        specs:
+          - https://identifiers.org/rrid/RRID:SCR_001209
+          - https://bio.tools/qualimap
+  DockerRequirement:
+    dockerPull: 'quay.io/biocontainers/qualimap:2.2.2d--1'
+requirements:
+  - class: ShellCommandRequirement
+
+# Base command
+baseCommand: [ "qualimap"]
+
+# Perform RNA-seq QC analysis on paired-end data http://qualimap.bioinfo.cipf.es/doc_html/command_line.html
+# Desired java memory size can be set using --java-mem-size option
+arguments:
+  - valueFrom: |-
+      rnaseq --paired --java-mem-size="$(inputs.javamem)"
+    shellQuote: false
```

Comment:
Why is `shellQuote` used here? I don't see any `|` or other IO redirection

Reply:
@mr-c - if I remove `shellQuote`, it adds a quote around the `valueFrom` as:
```
'rnaseq --paired --java-mem-size="4G"'
```

This causes the tool definition to fail with `ERROR: Unrecognized option: --java-mem-size="4G"` 

The run command (without `shellQuote`) that produces error is:

```
    /bin/sh \
    -c \
    'qualimap' 'rnaseq --paired --java-mem-size="4G"' '-a' 'proportional' '-gtf' 
    '/var/lib/cwl/stg5b3032a1-50be-4afd-a236-aa0080294216/ref-transcripts.gtf' 
    '-bam' '/var/lib/cwl/stg5149eaa5-dbda-4763-adb9-a7a5ad22163a/NA12878-ready_exome21.bam' 
    '-outdir' 'outQualimap' '-p' 'non-strand-specific'
```

And the one that is run successfully with `shellQuote: false` is 

```
    'qualimap' rnaseq --paired --java-mem-size="4G" '-a' 'proportional' '-gtf' 
    '/var/lib/cwl/stgba26c460-a8b7-463e-b4c5-760ec8581eac/ref-transcripts.gtf' 
    '-bam' '/var/lib/cwl/stgd39d7815-6bac-4f70-8ca5-16840e986434/NA12878-ready_exome21.bam' 
    '-outdir' 'outQualimap' '-p' 'non-strand-specific'
```